module/VuFind/src/VuFind/Controller/AbstractBase.php

```diff
@@ -136,6 +136,8 @@ protected function createViewModel($params = null)
             ->fromPost('layout', $this->params()->fromQuery('layout', false));
         if ('lightbox' === $layout) {
             $this->layout()->setTemplate('layout/lightbox');
+        }elseif(isset($params['layout']) && $params['layout']=='simple'){
```

Comment:
Is there really a need for this 'simple' layout? Is there a reason you can't use 'lightbox'? The only difference seems to be that the lightbox layout includes Piwik/Google Analytics tracking and simple does not. If tracking needs to be disabled for some reason, perhaps there is a way to do that without creating a whole new layout.

Reply:
I might be doing something wrong, but if I just use 'lightbox' as the layout in my action, it still renders the whole page instead of the html snippet that I am using in the getItemstatus JS to show the availability.

Reply:
The issue is that createViewModel is looking in the HTTP request parameters for the 'lightbox' flag, not in the parameters being sent to the view. For your situation, the best solution is probably to call `$this->layout()->setTemplate('layout/lightbox');` from your controller action instead of trying to pass an extra flag through to createViewModel. Does that work?

Reply:
Do you mean like this?  If so, I'm getting an error: resolver could not resolve to a file...
```PHP     
        $view = $this->createViewModel(
            [
              //'layout' => 'simple',
              'ids' => $ids,
              'action' => $action,
              'result' => $result,
            ]
        );
        //$view->setTemplate('RecordDriver/SolrOverdrive/status-full');
        $this->layout()->setTemplate('layout/lightbox');

        return $view; 
```

Reply:
Yes, that's correct -- but don't comment out the $view->setTemplate. You'll still need that. One is setting the view template (the "inside" content) and the other is setting the layout template (the "outside" wrapper).